meshroom/ui/qml/Viewer/Viewer2D.qml

```diff
@@ -990,18 +1043,18 @@ FocusScope {
                         }
 
                         ComboBox {
-                            id: imageType
+                            id: outputAttribute
                             property var activeNode: root.oiioPluginAvailable ? _reconstruction.activeNodes.get('allDepthMap').node : null
```

Comment:
Good point sorry about that ^^

This raises a question though : this _depthMapNodeName_ Label doesn't really make much sense now since we don't display just depth maps but any sequence of output images related to the viewpoints, and in addition users should be aware of what is being displayed since you actually need to double click a node to visualize it, so shouldn't we just remove that Label ?